src/models/timeoffRequests.js

```diff
@@ -0,0 +1,5 @@
+const db = require('./db/queries/timeoffRequests');
```

Comment:
Looks like this file is requiring `./db/queries/timeOffRequests`, but the path of the file added above is `db/queries/requests.js`? Seems like a bug?

Also can we get rid of the `queries` folder inside the `db` folder? I would rather have the path to be `src/models/db/timeoffRequests.js`